apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/BrazeTools/Modal/Body.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { Button, Flex, Input } from "@ledgerhq/react-ui";
+import { useGenerateLocalBraze } from "../Hooks/useGenerateLocalBraze";
+import { useTranslation } from "react-i18next";
+
+type TabKey = "NotificationContentCard" | "ActionContentCard" | "PortfolioContentCard";
+
+export const ModalBody: React.FC = () => {
+  const { t } = useTranslation();
+  const [formData, setFormData] = useState({
+    title: "",
```

Comment:
can we add default values here so we don't have to fill all the fields manually when we want to trigger a dummy test card ?